### DISCUSSION TITLE
Peaks_from_models with GQI

### DISCUSSION THREAD

**Discussion by staunton-golding** [dipy/dipy#3225] [Reconstruction] [answered]
If I want to get sh_coeff in a specific basis, can I plug the model instance created with GQI (created using GeneralizedQSamplingModel) into peaks_from_models? Or do the "models" mean something else in this case?

Best,
Hank

**Comment by skoudoro** [comment, marked as answer]
`GeneralizedQSamplingModel` instance can be plugged to  `peaks_from_models`

**Reply by staunton-golding** [reply]
Thank you!! Are there any models than can't be plugged into peaks_from_models? More context, we are using the tournier07 basis.

**Reply by skoudoro** [reply]
yes, the FreeWaterTensor model can not  or the IVIM model. I think MSDKI model can not also and I suppose I forget some other model but this is just to give you an idea.

**Reply by staunton-golding** [reply]
Thank you! Can FORECAST and MAP-MRI models be plugged in?

**Reply by skoudoro** [reply]
Yes for both model
